### ISSUE TITLE
Secure Sir 321

### ISSUE THREAD

**Issue by Flush1963** [hankolsen/com.securemeters.horstmann#36]
Hi hankolsen
Have you any plans to include support for theSecure Sir 321 at some later date.
Thanks
Bye
Damien

**Comment by hankolsen** [comment]
Hi @Flush1963!
I was not even aware of that device :)

I'll take a look during the weekend! 
Do you own a SIR 321 device? I don't have one myself so I will not be able to test and verify the functionality.

**Comment by Flush1963** [comment]
Yeah Ivown one brilliant for hot water tanks as they can switch 3kw
directly

**Comment by hankolsen** [comment]
@Flush1963, could you please try to install the test version available here: https://homey.app/sv-se/app/com.horstmann/Secure-Meters/test/

**Comment by hankolsen** [comment]
Merged into master from #38
